src/components/NavBar/nav-bar.scss

```diff
@@ -0,0 +1,4 @@
+.nav-bar{
+    z-index: 9999;
```

Comment:
actually nav-bar doesn't have any specified position, you want me to add `position:fixed` and make it sticky?